modules/swagger-codegen/src/main/resources/JavaSpring/apiController.mustache

```diff
@@ -20,24 +20,27 @@ import org.springframework.web.multipart.MultipartFile;
 
 import java.util.List;
 {{#async}}
-import java.util.concurrent.Callable;
-{{/async}}{{/java8}}
+    import java.util.concurrent.{{#java8}}Callable{{/java8}}{{^java8}}CompletableFuture{{/java8}};
+{{/async}}
+{{/java8}}
 
 {{>generatedAnnotation}}
 @Controller
 {{#operations}}
 public class {{classname}}Controller implements {{classname}} {
 {{^java8}}{{#operation}}
-    public {{#async}}Callable<{{/async}}ResponseEntity<{{>returnTypes}}>{{#async}}>{{/async}} {{operationId}}({{#allParams}}{{>queryParams}}{{>pathParams}}{{>headerParams}}{{>bodyParams}}{{>formParams}}{{#hasMore}},
+    public {{#async}}{{^java8}}Callable{{/java8}}{{#java8}}CompletableFuture{{/java8}}
```

Comment:
It's a "non-java8" block so no need to put java8 code here. Java8 default implementation is in the interface itself with default keyword